### ISSUE TITLE
Img2Img returns blank image

### ISSUE THREAD

**Issue by TrebledJ** [divamgupta/diffusionbee-stable-diffusion-ui#442]
Hi, I recently downloaded DB and was trying img2img, but all I'm getting is a blank image. I expected a fancy image. :(

<img width="840" alt="Screenshot 2023-06-24 at 3 24 33 AM" src="https://github.com/divamgupta/diffusionbee-stable-diffusion-ui/assets/39648915/792aecc7-4229-4388-aff1-03c27441e656">

DB Version: 1.7.4 (0024)
macOS Version: 12.6.6

**Comment by cristianae2001** [comment]
Hi, I’m experiencing the same issue. Did you figure out how to fix it?

**Comment by TrebledJ** [comment]
Yeah, I fixed it by uninstalling DiffusionBee and resorting to more manual work with e.g. https://github.com/basujindal/stable-diffusion.
